portfolio/src/main/webapp/addclass.html

```diff
@@ -0,0 +1,181 @@
+											<a class="nav-link drop" href="#">HOME</a>
+										</li>
+		
+										<li class="nav-item">
+											<a class="nav-link drop" href="#">CLASSES</a>
+										</li>
+		
+										<li class="nav-item">
+											<a class="nav-link drop" href="#">ABOUT</a>
+										</li>
+
+										<li class="nav-item">
+												<a class="nav-link drop" href="#">SIGN IN</a>
+											</li>
+									</ul>
+								</div>
+							</div>
+						</nav>
+					</div>
+		<!--Navigation Bar-->
+
+		<section class="form-header">
+      <div class="row">
+        <div class="col-10 offset-2 form-para">Add A Class</div>
+      </div>
+     
+      </div>
+
+      <div class="row">
+        
+
+        <div class="row">
+          <div class="col-3 offset-2 design-1"></div>
+        </div>
+      </div>
+    </section>
+
+    <form action="/class-handler" method="POST">
+      <div class="form-row">
+
+        <div class="form-group col-md-6 offset-2">
+          <label for="inputName" class = "input-header">Name</label>
+          <input type="text" class="form-control" id="inputName" placeholder="Name" name = "name">
+        </div>
+
+        <div class="form-group col-md-6 offset-2">
+          <label for="inputEmail" class = "input-header">Email</label>
+          <input type="email" class="form-control" id="inputEmail" placeholder="Email"  name = "email">
+        </div>
+    
+      </div>
+
+      <div class="form-group col-md-6 offset-2">
+        <label for="inputCollege" class = "input-header">College/University Name</label>
+        <input type="text" class="form-control" id="inputCollege" placeholder="College/University Name" name = "college" >
+      </div>
+      
+      <div class="form-group col-md-6 offset-2">
+        	<label for="inputDepartment" class = "input-header">Department</label>
+					<input type="text" class="form-control" id="inputDepartment" placeholder="Department" name = "department">
+			</div>
+			
+			<div class="form-group col-md-6 offset-2">
+        	<label for="inputProfessor" class = "input-header">Professor</label>
+					<input type="text" class="form-control" id="inputProfessor" placeholder="Professor" name = "professor">
+			</div>
+			
+			<div class="form-group col-md-6 offset-2">
+        	<label for="inputSemester" class = "input-header">Class Semester</label>
+					<input type="text" class="form-control" id="inputSemster" placeholder="Class Semester" name = "semester">
+			</div>
+
+			<div class="form-group col-md-6 offset-2">
+        	<label for="inputYear" class = "input-header">Class Year</label>
+					<input type="text" class="form-control" id="inputYear" placeholder="Class Year" name = "year">
+			</div>
+			
+			<div class="form-group col-md-6 offset-2">
+        	    <label for="inputClassNum" class = "input-header">Class Name/Number</label>
+				<input type="text" class="form-control" id="inputClassNum" placeholder="Class Name/Number" name = "classnum">
+			</div>
```

Comment:
since we are doing a form with each entry per row, how about having a dropdown section instead of manually input a class year and quarter? I think that way will make things easier and when we want to search for a certain class, we dont need to check for any blanks inputted by users. We can keep that for the next iteration